app/views/orders/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+    <h2>Order Summary</h2>
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>Product</th>
+          <th>Price</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @order.order_items.each do |product| %>
+        <tr>
+          <th scope="row">1</th>
+          <td><%="#{product.item.name} "%></td>
+          <td><%=" #{product.item.price} kr"%></td>
+        </tr>
+        <% end %>
+        <tr>
+          <th scope="row">Total:</th>
+          <td></td>
```

Comment:
Is this still here because you want to put something in this table cell?

Reply:
No, it is because we wanted some space between "Total price:" and "95 kr" for readability. See screenshot.

Reply:
Ok